Makefile

```diff
@@ -0,0 +1,52 @@
+
+py_sources = common/ daemons/ domain/ gateways/ handlers/ tests/ usecases/ utils/
+pytest_flags = -p no:warnings --cov-report=term --cov-report=html --cov-report=xml --cov=hathor
+mypy_flags = --warn-unused-configs --disallow-incomplete-defs --no-implicit-optional --warn-redundant-casts --warn-unused-ignores
+
+default:
+	echo "help"
```

Comment:
I don't this is quite it yet, when you run this rule this is what would happen:

```shell
$ make
echo help
help
```

What I would suggest is something like:

```make
.PHONY: help
help:
	@echo "Use 'make build' to generate the docker image and 'make run' to launch it."
```

Assuming those instructions are correct. Then when running this would be:

```shell
$ make
Use 'make build' to generate the docker image and 'make run' to launch it.
```